x-pack/metricbeat/module/panos/system/system.go

```diff
@@ -0,0 +1,152 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package system
+
```

Comment:
Same comments as I mentioned in vpn metricset. We can improve the error handling and refactor the Fetch and use errors.Join